core/src/main/java/org/elasticsearch/index/get/ShardGetService.java

```diff
@@ -238,7 +223,7 @@ public GetResult innerGet(String type, String id, String[] gFields, boolean real
                             value = source.source.length();
                         } else {
                             if (searchLookup == null) {
-                                searchLookup = new SearchLookup(mapperService, fieldDataService, new String[]{type});
+                                searchLookup = new SearchLookup(mapperService, null, new String[]{type});
```

Comment:
Good point, I was thinking about fielddata_fields but we can't get them anyway if a doc is only in the translog...